webpack/scenes/ContentViews/Create/ContentViewCreateSelectors.js

```diff
@@ -0,0 +1,18 @@
+import {
+  selectAPIStatus,
+  selectAPIError,
+  selectAPIResponse,
+} from 'foremanReact/redux/API/APISelectors';
+import { STATUS } from 'foremanReact/constants';
+import { CREATE_CONTENT_VIEW_KEY } from '../ContentViewsConstants';
+
+export const selectCreateContentViews = state =>
+  selectAPIResponse(state, CREATE_CONTENT_VIEW_KEY) || {};
+
+export const selectCreateContentViewStatus = state =>
+  selectAPIStatus(state, CREATE_CONTENT_VIEW_KEY) || STATUS.PENDING;
+
+export const selectCreateContentViewError = state =>
+  selectAPIError(state, CREATE_CONTENT_VIEW_KEY);
+
+export const selectIsCVSaving = state => state.katello?.contentViewCreate?.saving;
```

Comment:
You should be able to use the status selector to check this, we've been going with this recently over setting `saving` in a reducer.

Reply:
+1, I'd prefer checking the pending state with `selectAPIStatus` from `foremanReact/redux/API/APISelectors.js`

Reply:
Updated to use STATUS.PENDING